config/prod/dgutman-htan_s3.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: dgutman-htan_s3.yaml
+stack_name: dgutman-htan-dsa
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "dagutman@gmail.com"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) Synapse username (default ""), required if AllowWriteBucket=true
+  SynapseUserName: 'dagutman'
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+  # (Optional) true (default) to encrypt bucket, false for no encryption
```

Comment:
This and all `Optional` parameters  already have defaults.  You only need to specify them if you want to override the defaults.   Otherwise just remove them and the defaults will be set for you.